From 624b37db838a66ffe0193a0c39def185a7fd120b Mon Sep 17 00:00:00 2001
From: Tom Anderson <thomasanderson@chromium.org>
Date: Wed, 31 Jan 2018 18:15:06 +0000
Subject: [PATCH] Fix Gtk2 build

R=erg

Change-Id: I7707cf02b88a08e416d2fc0a17bfd75b918fe42c
Reviewed-on: https://chromium-review.googlesource.com/894993
Reviewed-by: Elliot Glaysher <erg@chromium.org>
Commit-Queue: Thomas Anderson <thomasanderson@chromium.org>
Cr-Commit-Position: refs/heads/master@{#533330}
---
 chrome/browser/ui/libgtkui/gtk_util.cc | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/chrome/browser/ui/libgtkui/gtk_util.cc b/chrome/browser/ui/libgtkui/gtk_util.cc
index fc6ffb9d2eb1..96f8c0e71472 100644
--- a/chrome/browser/ui/libgtkui/gtk_util.cc
+++ b/chrome/browser/ui/libgtkui/gtk_util.cc
@@ -56,14 +56,6 @@ void CommonInitFromCommandLine(const base::CommandLine& command_line,
   }
 }
 
-using GtkSetState = void (*)(GtkWidgetPath*, gint, GtkStateFlags);
-PROTECTED_MEMORY_SECTION base::ProtectedMemory<GtkSetState>
-    _gtk_widget_path_iter_set_state;
-
-using GtkSetObjectName = void (*)(GtkWidgetPath*, gint, const char*);
-PROTECTED_MEMORY_SECTION base::ProtectedMemory<GtkSetObjectName>
-    _gtk_widget_path_iter_set_object_name;
-
 }  // namespace
 
 namespace libgtkui {
@@ -245,6 +237,14 @@ float GetDeviceScaleFactor() {
   return linux_ui ? linux_ui->GetDeviceScaleFactor() : 1;
 }
 
+using GtkSetState = void (*)(GtkWidgetPath*, gint, GtkStateFlags);
+PROTECTED_MEMORY_SECTION base::ProtectedMemory<GtkSetState>
+    _gtk_widget_path_iter_set_state;
+
+using GtkSetObjectName = void (*)(GtkWidgetPath*, gint, const char*);
+PROTECTED_MEMORY_SECTION base::ProtectedMemory<GtkSetObjectName>
+    _gtk_widget_path_iter_set_object_name;
+
 }  // namespace
 
 void* GetGdkSharedLibrary() {
